checkov/gitlab_ci/checks/job/AvoidDoublePipelines.py

```diff
@@ -20,13 +22,14 @@ def __init__(self) -> None:
 
     def scan_conf(self, conf: dict[str, Any]) -> tuple[CheckResult, dict[str, Any]]:
         c = 0
+
         for rule in conf:
             if isinstance(rule, dict) and "if" in rule:
                 value = rule['if']
-                if value.startswith('$CI_PIPELINE_SOURCE'):
+                if value.startswith(PIPELINE_SOURCES):
                     c += 1
-                if c > 1:
-                    return CheckResult.FAILED, conf
+                    if c > 1:
+                        return CheckResult.FAILED, conf
```

Comment:
Why do we need to have `c`? Can't we just fail this check if the `if` returns `True`?
@eurogig 
```suggestion
                    return CheckResult.FAILED, conf
```